### PR TITLE
uboot-sunxi: Banana Pi M2 Ultra switch to mainline defconfig

### PR DESCRIPTION
**Pull request by abajk**
U-boot also have defconfig for this board. In 2019.01 branch they are identical.

Fixes: 26898a8f935f2c81fdbcf99a5508fcf455fd5615

Signed-off-by: Aleksander Jan Bajkowski <A.Bajkowski@stud.elka.pw.edu.pl>

### CODE REVIEW

File: package/boot/uboot-sunxi/Makefile [hunk 1]
```diff
@@ -294,7 +295,6 @@ UBOOT_TARGETS := \
 	pangolin \
 	pine64_plus \
 	Sinovoip_BPI_M2_Plus \
```
Comment (ynezz) [inline package/boot/uboot-sunxi/Makefile]: @CHKDSK88 BTW what about `Sinovoip_BPI_M2_Plus` ? Seems like it wont work these days.

```
ynezz@ntbk:/opt/devel/u-boot$ make Sinovoip_BPI_M2_Plus_config
***
*** Can't find default configuration "arch/../configs/Sinovoip_BPI_M2_Plus_defconfig"!
***
```

**Comment by ynezz** [comment]
> Fixes: 26898a8

I would remove this, that commit was probably fine as it just included that config in separate patch.

File: package/boot/uboot-sunxi/Makefile (hunk 1 above)
Comment (CHKDSK88) [reply package/boot/uboot-sunxi/Makefile]: @ynezz 

This is a reason: https://github.com/u-boot/u-boot/commit/268ae6548779ccd8ba38ce39d43f41be7e0bc133

It should be changed if we bump uboot-sunxi to >=2019.04.

**Comment by ynezz** [comment]
Thanks! Pulled into my staging tree at https://git.openwrt.org/openwrt/staging/ynezz.git

File: package/boot/uboot-sunxi/Makefile (hunk 1 above)
Comment (ynezz) [reply package/boot/uboot-sunxi/Makefile]: @CHKDSK88 Thanks, should be fixed via https://github.com/ynezz/openwrt/commit/fefa6f5a21a773c7fb434afe9f33012a4df7d815 as I plan to bump U-Boot to 2020.01 soon via https://github.com/ynezz/openwrt/commit/5cb06c0a62267bfb24f1a1c3d3fb6902d92db31b

Comment (CHKDSK88) [reply package/boot/uboot-sunxi/Makefile]: @ynezz 
Thanks for info. LGTM.
At this moment, I can't run test it because my banana was burned.

**Pull request closed without merging**